contracts/commitments/batched_fri_verifier.sol

```diff
@@ -801,9 +801,9 @@ library batched_fri_verifier {
         bytes calldata blob,
         types.fri_params_type memory fri_params, 
         types.fri_local_vars_type memory local_vars,
-        uint256 []memory xi
-    ) internal view returns(bool b){
-        uint256[7] memory precomputed = commitment_calc.eval2_precompute(fri_params.tmp_arr[0], xi[0], xi[1], fri_params.modulus);
+        uint256 [4]memory xi
```

Comment:
Are we sure that this always will be 4? Maybe, it makes sense to set a constant for this